Sources/SwiftyMonaco/MonacoViewController.swift

```diff
@@ -32,22 +32,48 @@ public class MonacoViewController: ViewController, WKUIDelegate, WKNavigationDel
         webView.layer?.backgroundColor = NSColor.clear.cgColor
         #endif
         view = webView
+        #if os(macOS)
+        DistributedNotificationCenter.default.addObserver(self, selector: #selector(interfaceModeChanged(sender:)), name: NSNotification.Name(rawValue: "AppleInterfaceThemeChangedNotification"), object: nil)
+        #endif
     }
     public override func viewDidLoad() {
         super.viewDidLoad()
         
+        loadMonaco()
+    }
+    
+    private func loadMonaco() {
         let myURL = Bundle.module.url(forResource: "index", withExtension: "html", subdirectory: "Resources")
         let myRequest = URLRequest(url: myURL!)
         webView.load(myRequest)
     }
```

Comment:
Monaco loading process is moved to another function while testing ways to change appearance in the editor. I think it's better than it was because now it's more readable than before.